scripts/v.in.wfs/v.in.wfs.py

```diff
@@ -130,12 +159,17 @@ def main():
         wfs_url += "&BBOX=" + bbox
 
     if flags["l"]:
-        wfs_url = options["url"] + "REQUEST=GetCapabilities&SERVICE=WFS&VERSION=1.0.0"
+        if options["version"]:
+            wfs_url += "&VERSION=" + options["version"]
```

Comment:
1. The `REQUEST=GetCapabilities&SERVICE=WFS` part got lost?
2. The version is always set because of the default answer, no need for `if ... else ...` here